mitosheet/mitosheet/tests/test_transpile.py

```diff
@@ -618,6 +618,33 @@ def test_transpile_fully_parameterized_function_string(tmp_path):
         f"{TAB}return import_dataframe_0, txt, Sheet1",
         "",
     ])
+    assert mito.mito_backend.param_metadata == [
+        {
+            'initial_value': 't',
+            'type': 'e',
+            'subtype': 's'
```

Comment:
What are these? This is strange if these are the actual values, b/c they feel like they shouldn't be.

Reply:
Can we also add so the `mito_test_wrapper` gets a `string` initial argument as well as a dataframe initial argument?

Reply:
(I know we don't let you parameterize it, but it's nice to include that current in the test)